src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -250,6 +256,11 @@ public abstract partial class JsonNamingPolicy
     public partial class JsonNode
     {
         public JsonNode() { }
+        public System.Text.Json.JsonElement AsJsonElement() { throw null; }
+        public static System.Text.Json.JsonNode DeepCopy(System.Text.Json.JsonNode jsonNode) { throw null; }
+        public static System.Text.Json.JsonNode GetNode(System.Text.Json.JsonElement jsonElement) { throw null; }
+        public static System.Text.Json.JsonNode Parse(string json) { throw null; }
```

Comment:
We probably want way more Parse overloads in here similar to the ones that JsonDocument supports.